docs/navigate.rst

```diff
@@ -198,6 +199,13 @@ View Node Dashboard
    c. **APIC Endpoint Details** - The Visibility dashboard displays APIC details for the Node: - 1. MAC 2. EPG 3. Node 4. Interface 5. VLAN Encap 
                                         
    d. **BIG-IP Endpoint Details** - The Visibility dashboard displays BIG-IP details for the Node: - 1. MAC 2. VLAN 3. Interfaces (There is a **View Logs** link beside interfaces to view the interface logs) 4. Self IPs
+   
+6. F5 ACI ServiceCenter supports VXLAN tunnel interfaces along with individual, Port channel, Virtual port channel interfaces on Node table visibility dashboard.
```

Comment:
....along with individual, Port channel, and Virtual port channel interfaces on the Node table visibility dashboard.